packages/main/src/ComboBox.js

```diff
@@ -839,9 +839,21 @@ class ComboBox extends UI5Element {
 			this.filterValue = this._selectedItemText;
 		}
 
+		if (isPhone() && event && event.target.getAttribute("icon") === "decline") {
```

Comment:
event.target.getAttribute("icon") === "decline" -> in my opinion it is not a reliable check. Imagine that we change the icon with another one - that check will not be valid anymore. If there is no way to distinguish that icon currently, it is better to add a specific class to it instead of checking its name. 
Another thing, if you find out a way to distinguish the icon, you do not have to check whether it is phone, is that icon is specific for a mobile device, because checking for the icon will return true only when you are on a mobile device and it is presented.